### ISSUE TITLE
Sara's email about people claiming data

### ISSUE THREAD

**Issue by trreynolds** [ldbase/ldbase_issues#157]
This github is for the email Sara sent to Brian about the below (and maybe what happens when you click):

The user *Cynthia U. Norris* has indicated that they believe they were entered as a contributor on *Dataset: Project KIDS Total Scores Data*

 As an administrator for this project, you may approve and connect the user to the item or reject it and leave things as they are.

 Go View Existing Records Requests to view the Existing Record Requests for this Project. https://urldefense.com/v3/__http://ldbase.org/group/10/existing-records-requests__;!!PhOWcWs!j7f5qWFiKYdpwtzEgWDpvkeFA1fvBMJfi3WlKkQBFzalSZCSfqqAu1eC7PY$

**Comment by barsenault-fsu** [comment]
I have made a change on Test to the message templates.  I had been putting in an HTML link which displays the title of the project and links to the URL, but the plain text email dismantles that and makes the URL look like a footnote.  Instead I changed the message templates so that the text includes the title and the URL as plain text.  So it will come though something like this:

![example-email](https://user-images.githubusercontent.com/54671134/132890253-fccf9338-55fd-4335-b03a-3e19fa2138e3.png)

Your email client will probably recognize that the URL is a URL and automatically turn it into a clickable link.  Some email systems may also add that "url defense" link around it too.  No problem it will still work.

That takes care of future messages and emails.  When viewing old messages in LDbase (My Messages) they will look mostly the same, but may have an extra piece of text like "@ldbase_object" before the link to the project or form.  That's the new placeholder in the template for the plain text name.  It wasn't in the template when those old messages were created so it can't replace it with a real value.  Otherwise the message is the same.
